development-tools/clang.rst

```diff
@@ -49,177 +54,92 @@ A complete list of sanitizers can be found at `Controlling Code Generation
 
 Clang and its sanitizers have strengths (and weaknesses). Its just one tool in
 the war chest to uncovering bugs and improving code quality. Clang should be
-used to compliment other methods, including Code Reviews, Valgrind, Coverity,
+used to complement other methods, including Code Reviews, Valgrind, Coverity,
 etc.
 
 Clang/LLVM setup
 ================
 
-This portion of the document covers downloading, building and installing Clang
-and LLVM. There are three components to download and build. They are the LLVM
-compiler, the compiler front end and the compiler runtime library.
-
-In preparation you should create a scratch directory. Also ensure you are using
-Python 2 and not Python 3. Python 3 will cause the build to fail.
-
-Download, build and install
----------------------------
-
-Perform the following to download, build and install the Clang/LLVM 3.4. ::
-
-    # Download
-    wget https://llvm.org/releases/3.4/llvm-3.4.src.tar.gz
-    wget https://llvm.org/releases/3.4/clang-3.4.src.tar.gz
-    wget https://llvm.org/releases/3.4/compiler-rt-3.4.src.tar.gz
-
-    # LLVM
-    tar xvf llvm-3.4.src.tar.gz
-    cd llvm-3.4/tools
+Pre-built Clang builds are available for most platforms:
 
-    # Clang Front End
-    tar xvf ../../clang-3.4.src.tar.gz
-    mv clang-3.4 clang
+- On macOS, Clang is the default compiler
+- For mainstream Linux distros, you can install a ``clang`` package
+- On Windows, the installer for Visual Studio (not Code)
+  includes the "C++ clang tools for windows" feature
```

Comment:
```suggestion
- On macOS, Clang is the default compiler.
- For mainstream Linux distros, you can install a ``clang`` package.
  In some cases, you also need to install ``llvm`` separately, otherwise
  some tools are not available.
- On Windows, the installer for Visual Studio (not Code)
  includes the "C++ clang tools for windows" feature.
```